AmplifyCombineSupport/AmplifyCombineSupport/AmplifyCombineSupport.h

```diff
@@ -0,0 +1,19 @@
+//
+//  AmplifyCombineSupport.h
+//  AmplifyCombineSupport
+//
+//  Created by Schmelter, Tim on 6/4/20.
+//  Copyright © 2020 Amazon Web Services. All rights reserved.
```

Comment:
what's this file used for? can we just remove it? also, the swiftformat didn't pick up the file to replace the header with the
```
//
// Copyright 2018-2020 Amazon.com,
// Inc. or its affiliates. All Rights Reserved.
//
// SPDX-License-Identifier: Apache-2.0
//
```